e2e/code_review_test.go

```diff
@@ -51,8 +51,8 @@ var _ = Describe("E2E TEST:CodeReview", func() {
 				Error:         nil,
 				Score:         checker.MaxResultScore,
 				NumberOfWarn:  0,
-				NumberOfInfo:  2,
-				NumberOfDebug: 30,
+				NumberOfInfo:  0,
```

Comment:
how about changing this tests to one of our test repos to avoid breaking? We can update it properly later.

Reply:
I would prefer leaving this in explicitly broken state so that we don't miss this when we are fixing e2e tests.

Reply:
there's one already https://github.com/ossf/scorecard/issues/783